### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/react-final-form ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations (optional). [[More info](https://github.com/OpenCollective/opencollective-cli)]
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

**Comment by erikras** [comment]
Wow, that `contributors.svg` is awesome!

**Comment by brieb** [comment]
Also want to thank you for your great contribution to the open source community -- final form is awesome! 🏁 

With this change, I get the following error when trying to `npm install react-final-form`

```
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! react-final-form@3.1.1 postinstall: `opencollective postinstall`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the react-final-form@3.1.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/breanna_bunge/.npm/_logs/2018-02-27T22_59_06_062Z-debug.log
```

Believe this is due to opencollective being a devDependency.

**Comment by monkeywithacupcake** [comment]
That is strange @brieb - you can remove the opencollective postinstall script if it is causing issues. It should show a little note about how the project is using opencollective and give users a link to donate.

File: package.json [hunk 1]
```diff
@@ -49,6 +50,7 @@
     "lint-staged": "^6.0.1",
     "nps": "^5.7.1",
     "nps-utils": "^1.5.0",
+    "opencollective": "^1.0.3",
```
Comment (oliviertassinari) [inline package.json]: It shouldn't be a dev dependency.

**Pull request merged**